.github/workflows/release.yaml

```diff
@@ -27,4 +27,47 @@ jobs:
           args: release --rm-dist
           workdir: ./cmd/manager/ # <-- path to the directory containing the main() function
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  container-images:
+    name: Build and push container images
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_REGISTRY: quay.io
+      DOCKER_REPOSITORY: integreatly/grafana-operator
```

Comment:
```suggestion
      DOCKER_REPOSITORY: grafana-operator/grafana-operator
```

Reply:
We've migrated away from integreatly, on top of that we no longer have permissions to push to that repo due to some access settings that have changed since our last release.
@pb82 I believe you'll have to generate a new token secret for this workflow to be able to push to our new repo.